src/lib.rs

```diff
@@ -251,7 +249,8 @@ pub use storage::{MergeError, PackedData};
 /// Please note that you should call `World::maintain`
 /// after creating / deleting entities with this resource.
 ///
-/// When `.join`ing on `Entities`, you will need to do it like this:
+/// When `.join`ing on `Entities`, you will need to do it like this
```

Comment:
"When joining `Entities` you will need to first dereference `Entities`/`Fetch<EntitiesRes>` to get the underlying `EntitiesRes`,  then you will need to re-reference it since only the referenced `Entities` has an implementation of `Join`."